src/main/resources/data.sql

```diff
@@ -10,4 +10,6 @@ INSERT INTO tutor (first_name,last_name,email) VALUES ('Zach','Sisco','zachsisco
 INSERT INTO tutor (first_name,last_name,email) VALUES ('Yinon','Rousso','yinonRousso@example.org');
 INSERT INTO tutor (first_name,last_name,email) VALUES ('Kate','Perkins','kateperkins@example.org');
 INSERT INTO tutor (first_name,last_name,email) VALUES ('George','Kripac','Georgekripac@example.org');
+INSERT INTO tutor (first_name,last_name,email) VALUES ('Aaron', 'Huang', 'huang.aaron185@gmail.com');
```

Comment:
Great to add more seed data, but if you do not want your email to live in this repository indefinitely, feel free to remove it.